### PR TITLE
CSF Standards: ProgressBox conditionally shows lesson number

### PR DESCRIPTION
**Pull request by Erin007**
[LP-961](https://codedotorg.atlassian.net/browse/LP-961)
[spec](https://docs.google.com/document/d/1xlm_syljGXIapl72GaKxN63V7kt6syHyvWHuqZhTCfE/edit#)

In preparation for CSF standards work, I extended `ProgressBox` to conditionally show a lesson number. 

Screen Shot 2019-12-10 at 3 18 44 PM

### CODE REVIEW

File: apps/src/templates/sectionProgress/ProgressBox.jsx [hunk 1]
```diff
@@ -11,7 +11,13 @@ const styles = {
     boxSizing: 'content-box'
   },
   filler: {
-    width: 20
+    width: 20,
+    position: 'absolute'
+  },
+  lessonNumber: {
```
Comment (dmcavoy) [inline apps/src/templates/sectionProgress/ProgressBox.jsx:17]: I pulled this in to try it out with the Lesson Status Dialog and with just one number for the lesson number it does not center the number.

Comment (Erin007) [reply apps/src/templates/sectionProgress/ProgressBox.jsx:17]: Good catch!  I was focused on wide 2-digit numbers being too big, I forgot to think about the singles. Updating...

**Comment by Erin007** [comment]
new styling to center numbers:
Screen Shot 2019-12-11 at 2 27 48 PM

**Review by dmcavoy — APPROVED** [review]
Looks good! Thanks for updating!

**Pull request merged**